src/runtime/crt/host/microtvm_api_server.py

```diff
@@ -35,6 +37,18 @@
 
 IS_TEMPLATE = not os.path.exists(os.path.join(PROJECT_DIR, MODEL_LIBRARY_FORMAT_RELPATH))
 
+MEMORY_SIZE_BYTES = 2 * 1024 * 1024
```

Comment:
nit: maybe add comment as to why this is default memory size

Reply:
added, it's not an interesting reason. It's basically chosen to pass CRT tests in TVM

Reply:
@mehrdadh I think that's actually interesting / important info: the constant is determined experimentally, it really good to have comment about it as @alanmacd requested, I don't even consider it a nit :-)